src/pages/Admin/Admin.tsx

```diff
@@ -125,6 +125,9 @@ const VenueList: React.FC<VenueListProps> = ({
         <Link to="/admin/venue/creation" className="btn btn-primary">
           Create a venue
         </Link>
+        <a href={SPARKLEVERSE_ITY} target="_blank" rel="noreferrer">
+          SparkleVersity Tutorials & Creator Guides
+        </a>
```

Comment:
```suggestion
        <a href={SPARKLEVERSE_ITY} {...getExtraLinkProps(true)}>
          SparkleVersity Tutorials & Creator Guides
        </a>
```

Reply:
@goran-peoski-work fixed in e4059b28059129a2c241deb31894acd1d0feac1a